manifests/plugin_bands.pp

```diff
@@ -6,7 +6,15 @@
   $git_branch = "master",
 
   # INI settings below
-  $stage_agreement_deadline = undef,
+  $auction_start = "2016-02-21 11",
+  $band_panel_deadline = "2015-12-20",
+  $band_bio_deadline = "2016-01-03",
+  $band_agreement_deadline = "2016-01-17",
+  $band_w9_deadline = "2016-01-17",
+  $band_merch_deadline = "2016-01-31",
+  $band_charity_deadline = "2016-01-31",
+  $band_badge_deadline = "2016-02-07",
+  $stage_agreement_deadline = "2016-02-07",
```

Comment:
see comment below, I think we can make it cleaner without having to resort to having hardcoded dates all over the place.  I also consider the dates in configspec.ini something we should move away from, prioritized for plugins that are going to be used in multiple event ubers.

or, if we do have to have default dates in some places, it would be good to have nonsensical, clearly obviously placeholder dates like June 6, 2222 or similar